src/routes.php

```diff
@@ -124,7 +124,7 @@
 		));
 
 		//CSRF protection in forms
-		Route::group(array('before' => 'csrf'), function()
+		Route::group(array('middleware' => 'csrf'), function()
```

Comment:
It is necessary to include the fullpath to the CSRF middleware here as Laravel doesn't add the `csrf` key by default or tell the user to add the `csrf` key in the `app/Http/Kernel.php` file.

```php
    Route::group(array('middleware' => 'App\Http\Middleware\VerifyCsrfToken'), function()
```